variables.tf

```diff
@@ -284,7 +284,7 @@ variable "function_app_storage_account_container_prefix" {
 variable "function_app_version" {
   type = string
   description = "Function app code version (hash)"
-  default = "5bdd751e25fb740de4e967763a547f28"
+  default = "ce8cbfcfe64e223f7b56f151b8af2d59"
```

Comment:
why do we have this hash as variable?

Reply:
used for `function_app_zip_name            = "${var.function_app_dist}/${var.function_app_version}.zip"`
and then:
`WEBSITE_RUN_FROM_PACKAGE = "https://${local.weka_sa}.blob.core.windows.net/${local.weka_sa_container}/${local.function_app_zip_name}"`

also we validate:
```
  lifecycle {
    precondition {
      condition     = var.function_app_version == local.function_app_code_hash
      error_message = "Please update function app code version."
    }
    ignore_changes = [site_config, tags]
  }
  ```
  It is important since we might update the code and not distribute it, i.e. it will not really be used.